unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/CharacterPreviewController/CharacterPreviewFactory.cs

```diff
@@ -0,0 +1,38 @@
+using DCL;
+using System;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+public class CharacterPreviewFactory : ICharacterPreviewFactory
```

Comment:
can you move it in the namespace, please?

Reply:
I will move them to namespaces, it is just like this because there were no namespaces for older classes.